src/app/components/dashboard/exoft-achievements/exoft-achievements.component.html

```diff
@@ -5,8 +5,8 @@ <h4>EXOFT ACHIEVEMENTS</h4>
     <div class="amount">80</div>
   </div>
 
-  <div class="container-content" *ngFor='let user of users'>
-    <div class="ach-content">
+  <div class="container-content">
+    <div class="ach-content" *ngFor='let user of users'>
 
       <div class="ach-photo-name-time">
         <div class="pic" style="background-image: url({{user.photo}});"></div>
```

Comment:
for dynamic variable in styles use ngStyle, always try to omit inline styles if possible